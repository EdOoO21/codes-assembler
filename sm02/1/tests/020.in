405 611 569 802 852 -691 -468 380 301 739 853 -793 786 -330 173 854 -654 -945 -158 -167 -847 -789 -744 -347 -29 945 189 -80 -157 -573 -591 -351 280 860 395 -313 -327 -125 -811 285 60 649 723 7 -171 -834 -578 173 -504 -929 -587 -807 -829 -611 -481 420 565 -420 -376 413 -475 -666 277 -759 854 -953 -453 445 -521 759 722 -551 175 -514 -886 -974 636 320 264 -385 -428 -720 422 324 -313 228 477 -84 856 -687 889 307
