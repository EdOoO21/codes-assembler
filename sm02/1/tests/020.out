307
889
-687
856
-84
477
228
-313
324
422
-720
-428
-385
264
320
636
-974
-886
-514
175
-551
722
759
-521
445
-453
-953
854
-759
277
-666
-475
413
-376
-420
565
420
-481
-611
-829
-807
-587
-929
-504
173
-578
-834
-171
7
723
649
60
285
-811
-125
-327
-313
395
860
280
-351
-591
-573
-157
-80
189
945
-29
-347
-744
-789
-847
-167
-158
-945
-654
854
173
-330
786
-793
853
739
301
380
-468
-691
852
802
569
611
405
